packages/mmseg/mmseg.0.1.1/descr

```diff
@@ -0,0 +1,3 @@
+MMSEG
```

Comment:
This short description is not very helpful to someone browsing a listing of names + short descriptions.